lib/LedgerSMB/Template/LaTeX.pm

```diff
@@ -57,13 +57,14 @@ use strict;
 use Template::Latex;
 use Template::Parser;
 use LedgerSMB::Template::TTI18N;
+use LedgerSMB::Magic qw( UNI_aring UNI_Aring );
 use Log::Log4perl;
 use TeX::Encode::charmap;
 use TeX::Encode;
 
 BEGIN {
-    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(0x00c5)};
-    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(0x00e5)};
+    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(UNI_Aring)};
+    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(UNI_aring)};
```

Comment:
Hmm. Why are we deleting these specific chars?
At a minimum we need some comments as to why it's necessary.
@ehuelsmann and @eihnverfr can one of you cast some light on this?